vipps-recurring-api-faq.md

```diff
@@ -39,6 +41,15 @@ Document version: 1.5.1.
 * [Invoicing](#invoicing)
 * [Questions?](#questions)
 
+## Does Vipps automatically create charges for an agreement?
+ 
+ No. Vipps does _not_ create charges based on the agreement, this is left up to the merchant to create.  
+ When a merchant creates a charge, Vipps will actually attempt to charge the customer.  
+ If the merchant doesn't create a charge, nothing will be done.
```

Comment:
Mention the retryDays?

"When a merchant creates a charge, Vipps will actually attempt to charge the customer, starting on the `due` date and for as long s specified in `retryDays`"

Add link: https://github.com/vippsas/vipps-recurring-api/blob/master/vipps-recurring-api.md#call-by-call-guide